src/components/Skeleton/Skeleton.tsx

```diff
@@ -17,16 +17,22 @@ const wave = keyframes`
     transform: translateX(100%);
   }
 `;
+
 const Skeleton = styled.span.withConfig({
   shouldForwardProp: (property) => !includes(property, ['width']),
-})<{ width?: number | string }>`
+})<{
+  width?: number | string;
+  height: number | string;
+  variant: 'circular' | 'text';
+}>`
   display: block;
   background: ${getColor('neutral.300')};
   position: relative;
   overflow: hidden;
-  height: 1.25rem;
+  height: ${({ height }) => pxToRem(height ?? '1.25rem')};
```

Comment:
I didn't implement it for the Datatable but it would be nice to have the size inferring the same as MUI Skeleton use https://mui.com/material-ui/react-skeleton/#inferring-dimensions